docs/2.getting-started/2.getting-started.mdx

```diff
@@ -0,0 +1,58 @@
+---
+name: Getting Started in 5 Steps
+route: /getting-started/
+---
+
+import { NextButton } from '../components/nextButton.tsx'
+import { PrevButton } from '../components/prevButton.tsx'
+
+# Getting Started in 5 Steps
+
+Let's create our first Botonic bot!
+
+## Step 1 – Install the Botonic Command Line Interface
+1. Install the LTS v12 of [NodeJS installer](https://nodejs.org/). You can also install Yarn.
+2. Verify the installation version by running `npm --version` and `node --version`. **Note:** You must have at least `node >= 10.0.0` version installed.
+3. Perform a global installation by running  `sudo npm install -g @botonic/cli` (Mac/Linux) or `npm install -g @botonic/cli` (as admin on Windows).
+4. Wait until the CLI is installed.
+
+## Step 2 – Create a Bot
+1. Run `botonic new <bot_name>` 
+2. Select one of the [templates](/templates/templates) available. 
+
+Congratulations: Your bot is created!
+
+> #### NOTE: 
+> You can optionally pass the name of the template as a second argument such as `botonic new <bot_name> tutorial`
+
+## Step 3 – Test Your Bot
+1. Set yourself in the directory you created: `cd <bot_name>`.
+2. Test your bot in your browser by running `botonic serve` (Mac/Linux) or `npm run start` (Windows - For more details, run `npm run build`).
+3. Copy the `localhost:8080` URL and paste it in your browser.
+4. Try chatting with your bot!
+> #### NOTE: 
+> If you started with the "blank" template, the bot answers with "I don't understand you".
+
+## Step 4 – Deploy Your Bot 
+
+1. From your command line, run `botonic deploy`.
+2. Enter your credentials (email/password) if you already signed up at hubtype.com or create a new account as recommended. 
```

Comment:
hubtype.com --> `[app.hubtype.com](https://app.hubtype.com/login)` or like the link just below (in step 5 section)